### PR TITLE
Attempt at actions

### PR DESCRIPTION
**Pull request by local-minimum**
Resolves #1 

# Introduces 6 different CI tests
- Front-end linting, expected outcome  ❌ (some lint errors) 
- Front-end testing, expected outcome ❌ (not sure why images look different now, but they do, perhaps because it's run in chrome headless instead of chrome)
- Front-end build, expected outcome  ✅ 
- Back-end lint and types, expected outcome ❌ (maaany lint errors and mypy wont work until py 3.5+)
- Back-end tests, expected outcome ❌ (guessing this is part of issues with 2.7 installs so don't want to spend time one it)
- Docker build, expected outcome ✅ 

# Adds templates for issues

The usual

# Additional

- Have added protection and configured the repo some
- Created new issue for future publication of docker images

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Ipsum lorem..
+
+# 🔢 To reproduce
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+# ✌🏽 Expected behavior
```
Comment (skymandr) [inline .github/ISSUE_TEMPLATE/bug.md:21]:
```suggestion
# ✌🏽 Expected behaviour
```
;)

File: Dockerfile [hunk 2]
```diff
@@ -4,13 +4,13 @@ WORKDIR /src
...
+FROM ubuntu:18.04
 RUN apt update && apt -y install usbutils software-properties-common python-pip
 # net-tools & iputils-ping are used in the xml-writer which should be removed soon
 RUN apt -y install net-tools iputils-ping
-RUN echo -e "\n" | add-apt-repository ppa:rolfbensch/sane-release
-RUN apt-get update
-RUN apt -y install libsane=1.0.27-xenial1 sane-utils=1.0.27-xenial1 libsane-common=1.0.27-xenial1
+# RUN echo -e "\n" | add-apt-repository ppa:rolfbensch/sane-release
+# RUN apt-get update
+RUN apt -y install libsane sane-utils libsane-common
```
Comment (skymandr) [inline Dockerfile:13]: Issue på detta?

File: tox.ini [hunk 3]
```diff
@@ -24,3 +24,24 @@ commands =
         --cov scanomatic --cov scripts --cov-report xml \
         --junitxml result.xml --ignore dev \
         {posargs}
+
+[testenv:mypy]
+basepython = python2.7
```
Comment (skymandr) [inline tox.ini:29]: visst känns det lite jobbigt, även om det är övergående..? ;)

Comment (local-minimum) [reply tox.ini:29]: jo väldigt

File: Dockerfile (hunk 2 above)
Comment (local-minimum) [reply Dockerfile]: jag hade glömt att pusha... men jag anser att det ingår i migrera till 3.9ish att se till att det faktiskt funkar att scanna.

**Pull request merged**